### ISSUE TITLE
Do you have any plan for 1.13?

### ISSUE THREAD

**Issue by a3102397** [Ko0z/zUI#11]

**Comment by Ko0z** [comment]
Yes, since I were not in the beta I could not get it done prior to launch. I have been playing like everyone else and haven't had the time to spare for addon development. I will port the addon for classic as soon as the launch heat breaks down a little. I hope you guys understand.

**Comment by Macumbafeh** [comment]
Hey did you do it for Classic? i'm interested for both Classic and TBC Classic

**Comment by Ko0z** [comment]
Sorry this never happened
